include/gzp.h

```diff
@@ -15,7 +15,11 @@
 #include <stdint.h>
 #include <stdbool.h>
 #include <nrf_gzll.h>
+#if defined(GAZELL_PAIRING_USER_CONFIG_ENABLE)
```

Comment:
Missing `CONFIG_` prefix when referring to Kconfig options in the code. Please fix, otherwise it would not work (seems like it was accidentally broken in the recent force push)

Reply:
Fixed